lib/main.dart

```diff
@@ -25,7 +26,7 @@ class MyApp extends StatelessWidget {
       darkTheme: Theming.darkTheme,
       initialRoute: '/',
       routes: {
-        '/': (context) => SplashScreen(),
+        '/': (context) => DrawerScreen(),
```

Comment:
If you have changed the initialRoute from the SplashScreen()  to the DrawerScreen() for debug purpose then before giving your pull request please change it to the previous one .